xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataPagination.vue

```diff
@@ -28,11 +28,12 @@
 -->
 <template>
   <!-- Pagination -->
-  <nav class="livedata-pagination">
+  <nav class="livedata-pagination"
+    :aria-label="$t('livedata.pagination.label', [this.data.query.source.className])">
```

Comment:
There is no reason why a source should have a `className`, this is specific to the live table source and also there it isn't mandatory.